cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/CompileCommand.java

```diff
@@ -211,8 +211,8 @@ public void execute() {
         
             targetPath = this.sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
-            throw LauncherUtils.createLauncherException("invalid ballerina source path, it should either be a module " +
-                                                        "name in a ballerina project or a file with a \'" +
+            throw LauncherUtils.createLauncherException("This is an invalid Ballerina source path. It should either be a module " +
```

Comment:
Same comment as above, since the path isn't mentioned.

Reply:
Changed to: